plugins/InspectionGadgets/testsrc/com/siyeh/ig/logging/PlaceholderCountMatchesArgumentCountInspectionTest.java

```diff
@@ -35,6 +38,14 @@ protected LocalInspectionTool getInspection() {
       "package org.apache.logging.log4j.util;" +
       "public interface Supplier<T> {" +
       "    T get();" +
+      "}",
+
+      "package org.apache.logging.log4j;" +
+      "import org.apache.logging.log4j.util.Supplier;" +
+      "public interface LogBuilder {" +
+      "  public void log(String format, Object p0);" +
```

Comment:
This overload is not yet in the released version, but has been merged and will be in the the next version. See https://github.com/apache/logging-log4j2/commit/6913465c030ce02b3d6abb5a11b31f5f392f6b2a